tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
```

Comment:
Again unrelated to your PR.

Reply:
Same as above; I will remove into a different PR once other code parts look good.